mozilla_schema_generator/glean_ping.py

```diff
@@ -13,20 +13,57 @@
 class GleanPing(GenericPing):
 
     schema_url = "https://raw.githubusercontent.com/mozilla-services/mozilla-pipeline-schemas/dev/schemas/glean/baseline/baseline.1.schema.json" # noqa E501
-    probes_url = "https://probeinfo.telemetry.mozilla.org/glean/{}/metrics"
+    probes_url_template = "https://probeinfo.telemetry.mozilla.org/glean/{}/metrics"
     repos_url = "https://probeinfo.telemetry.mozilla.org/glean/repositories"
 
-    default_probes_url = probes_url.format("glean")
+    default_dependencies = ['glean']
     default_pings = {"baseline", "events", "metrics"}
     ignore_pings = {"default", "glean_ping_info", "glean_client_info"}
 
     def __init__(self, repo):  # TODO: Make env-url optional
-        super().__init__(self.schema_url, self.schema_url, self.probes_url.format(repo))
+        self.repo = repo
+        super().__init__(
+            self.schema_url,
+            self.schema_url,
+            self.probes_url_template.format(repo)
+        )
+
+    def get_dependencies(self):
+        # Get all of the library dependencies for the application that
+        # are also known about in the repositories file.
+
+        # The dependencies are specified using library names, but we need to
+        # map those back to the name of the repository in the repository file.
+
+        dependencies = self._get_json(self.dependencies_url.format(self.repo))
```

Comment:
Where is `dependencies_url` set?

Reply:
Also, we may want to call it `dependencies_url_template` to match the `probes_url_template`

Reply:
Ah -- I thought this was running, but it wasn't, since I was running the wrong command (small glitch in the docs, now fixed here).